### PR TITLE
refactor: drop modules inside addon-charts

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/demo/src/modules/components/legend-item/examples/2/index.ts [hunk 1]
```diff
@@ -1,17 +1,34 @@
...
+    TuiAlertService,
+    tuiFormatNumber,
+    TuiNotificationComponent,
+    TuiSvgComponent,
+} from '@taiga-ui/core';
 
 @Component({
-    selector: 'tui-legend-item-example-2',
+    standalone: true,
+    imports: [
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/legend-item/examples/2/index.ts:17]: Add checkbox import to fix screenshots

image

Comment (splincode) [reply projects/demo/src/modules/components/legend-item/examples/2/index.ts:17]: thank you

Comment (splincode) [reply projects/demo/src/modules/components/legend-item/examples/2/index.ts:18]: fixed

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/line-chart/index.html [hunk 2]
```diff
@@ -7,31 +7,29 @@
...
+        />
 
         <tui-doc-example
             id="hint"
             heading="Hint"
-            [content]="example4"
+            [component]="4 | tuiComponent"
+            [content]="4 | tuiExample"
         >
             <tui-notification class="tui-space_bottom-6">
```
Comment (waterplea) [inline projects/demo/src/modules/components/line-chart/index.html:34]: I don't think we need it now that all examples are standalone. We can also do ngFor over the examples

File: projects/demo/src/modules/components/line-days-chart/examples/2/pipe.ts [hunk 3]
```diff
@@ -15,6 +15,7 @@ function even<T>(array: readonly T[]): readonly T[] {
 }
 
 @Pipe({
+    standalone: true,
```
Comment (waterplea) [inline projects/demo/src/modules/components/line-days-chart/examples/2/pipe.ts:18]: Maybe we should move this class to `index.ts` so we can use our regular dynamic examples?

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/line-chart/index.html (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/components/line-chart/index.html]: fixed

File: projects/demo/src/modules/components/line-days-chart/examples/2/pipe.ts (hunk 3 above)
Comment (splincode) [reply projects/demo/src/modules/components/line-days-chart/examples/2/pipe.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Pull request merged**